.github/workflows/deploy_MainNet_UI.yml

```diff
@@ -22,12 +22,12 @@ jobs:
         run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
 
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
```

Comment:
should be 
`uses: actions/checkout@163217dfcd28294438ea1c1c149cfaf66eec283e # v4.2.1`